app/editor/src/features/reports/av-overview/OverviewGrid.tsx

```diff
@@ -64,20 +67,20 @@ export const OverviewGrid: React.FC<IOverviewGridProps> = ({ editable = true, in
       findContent({
         seriesId: values.sections[index].seriesId,
         publishedStartOn: !!values.sections[index].startTime
-          ? moment()
+          ? moment(queryDate)
```

Comment:
this just pulls the queryDate from the url instead of just using today's date

so now it matches the selected date and will find clips from the selected date